R/translate.R

```diff
@@ -63,17 +63,71 @@ restore_cdata <- function(x) {
   x
 }
 
+# picking some popular languages from https://confluence.atlassian.com/doc/code-block-macro-139390.html
+supported_languages_default <- c(
```

Comment:
Why putting only three of them ? 
I understand Rmarkdown won't be used for all those language though... 
I would add in the default **Bash**, **CSS**, **HTML**, **JavaScript**, **YAML**
I already used those ones in Rmd document, mainly for documentation purposes. 

The extra supported languages mechanism will allow to add those, so feel free to keep the defaut you think are best.

Reply:
Thanks, agreed!